### PR TITLE
EDU-403 Change links styling

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-403

### CODE REVIEW

File: src/components/account-tab.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ function AccountTab({ formItemLayout, tailFormItemLayout }) {
       <UsernameFormItem name="username" usernamesInUse={state.forbiddenUsernames} initialValue={user.username} {...formItemLayout} />
       <EmailFormItem name="email" emailsInUse={state.forbiddenEmails} initialValue={user.email} {...formItemLayout} />
       <FormItem {...tailFormItemLayout}>
-        <Button type="link" size="small" onClick={handleResetPasswordClick}>{t('resetPassword')}</Button>
+        <a onClick={handleResetPasswordClick}>{t('resetPassword')}</a>
```
Comment (ahelmberger) [inline src/components/account-tab.js:80]: Just out of curiosity, why did you change the Button into an a?

**Review by ahelmberger — APPROVED** [review]

File: src/components/account-tab.js (hunk 1 above)
Comment (adrianaluchian) [reply src/components/account-tab.js:80]: I was clicking through the pages to make sure nothing is broken or looks very bad after this styling change and I noticed that this guy was the only one that was not restyled as a link (had no underline). So I streamlined it.
